app/frontend/src/search/geoloc.js

```diff
@@ -0,0 +1,7 @@
+/* global fetch */
+
+export const getCoordinates = query => {
+    return fetch(`https://localhost:3000/api/v1/coordinates/${query}`).then(data => {
```

Comment:
You've got a hardcoded dev URL here...

Reply:
IE requires a polyfill for `fetch` I believe: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API.  You might be better off packaging [axios](https://github.com/axios/axios)

Reply:
probably go for axios and yes i agree about the 'should' thing - force of habit